sys/net/if_bridge.c

```diff
@@ -2609,6 +2609,12 @@ bridge_input(struct ifnet *ifp, struct mbuf *m)
 			sc->sc_if_input(bifp, mc2);
 		}
 
+		/* Do not process none-local-targeted packets */
```

Comment:
You've got a whitespace issue here too.

It's got two spaces followed by a tab, where it just wants to be the tab.

Reply:
And there's missing whitespace between 'if' and '(!...'.